src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py

```diff
@@ -81,7 +81,7 @@ def register_providers():
     _register_one_provider("Microsoft.MachineLearningServices/registries", '2022-10-01-preview', True)
     _register_one_provider('Microsoft.DBforMySQL/flexibleServers', '2022-09-30-privatepreview', False)
     _register_one_provider("Microsoft.DocumentDB/mongoClusters", '2023-03-01-preview', True)
-
+    _register_one_provider('Microsoft.DBforPostgreSQL/flexibleServers', '2023-06-01-preview', False)
 
```

Comment:
You should leave 2 empty lines here. It is custom to have 2 empty lines between function definitions in python code.

Reply:
this should also be fixed in the test source file.

Reply:
Added 2 blank lines between each function definitions.